op-service/eth/output_test.go

```diff
@@ -32,3 +34,19 @@ func TestOutputV0Codec(t *testing.T) {
 	_, err = UnmarshalOutput([]byte{64: 0xA})
 	require.ErrorIs(t, err, ErrInvalidOutput)
 }
+
+func TestOutputJsonMarshal(t *testing.T) {
+	output := OutputV0{
+		StateRoot:                Bytes32{1, 2, 3},
+		MessagePasserStorageRoot: Bytes32{4, 5, 6},
+		BlockHash:                common.Hash{7, 8, 9},
+	}
+	jsonOutput, _ := json.Marshal(output)
+	expectedJson, err := os.ReadFile("testdata/output_v0.json")
+	require.NoError(t, err)
+	require.JSONEq(t, string(expectedJson), string(jsonOutput))
+
+	var unmarshaled OutputV0
+	json.Unmarshal([]byte(expectedJson), &unmarshaled)
```

Comment:
Lint is failing here:
```
op-service/eth/output_test.go:50:16: Error return value of `json.Unmarshal` is not checked (errcheck)
        json.Unmarshal([]byte(expectedJson), &unmarshaled)
```

```suggestion
	require.NoError(t, json.Unmarshal([]byte(expectedJson), &unmarshaled))
```

Reply:
@ajsutton fixed it.